jest.config.js

```diff
@@ -0,0 +1,18 @@
+// See https://jestjs.io/docs/en/configuration.
+module.exports = {
```

Comment:
As we discussed this will need updating to be based on the existing Jest config in the `package.json`.